app/models.py

```diff
@@ -2038,18 +2034,19 @@ class DateTimeDimension(db.Model):
 class FactNotificationStatus(db.Model):
     __tablename__ = 'ft_notification_status'
 
-    bst_date = db.Column(db.Date, index=True, primary_key=True, nullable=False)
-    template_id = db.Column(UUID(as_uuid=True), primary_key=True, index=True, nullable=False)
+    bst_date = db.Column(db.Date, index=True, primary_key=True, nullable=False, default=datetime.datetime.utcnow)
+    template_id = db.Column(UUID(as_uuid=True), primary_key=True, index=True, nullable=False, default=uuid.uuid4)
     service_id = db.Column(
         UUID(as_uuid=True),
         primary_key=True,
         index=True,
         nullable=False,
+        default=uuid.uuid4,
     )
-    job_id = db.Column(UUID(as_uuid=True), primary_key=True, index=True, nullable=False)
-    notification_type = db.Column(db.Text, primary_key=True, nullable=False)
-    key_type = db.Column(db.Text, primary_key=True, nullable=False)
-    notification_status = db.Column(db.Text, primary_key=True, nullable=False)
+    job_id = db.Column(UUID(as_uuid=True), primary_key=True, index=True, nullable=False, default=uuid.uuid4)
+    notification_type = db.Column(db.Text, primary_key=True, nullable=False, default=SMS_TYPE)
+    key_type = db.Column(db.Text, primary_key=True, nullable=False, default=KEY_TYPE_NORMAL)
+    notification_status = db.Column(db.Text, primary_key=True, nullable=False, default=NOTIFICATION_DELIVERED)
```

Comment:
```suggestion
    notification_status = db.Column(db.Text, primary_key=True, nullable=False, default=NOTIFICATION_CREATED)
```
I think this should default to "created" not "delivered" just in case.

Reply:
Agreed, updating